vmdb/app/presenters/menu/default_menu.rb

```diff
@@ -34,25 +34,34 @@ def clouds_menu_section
       end
 
       def infrastructure_menu_section
+        def hybrid_infra_name(name1, name2, name3)
+          lambda do
+            case EmsCluster.node_types
```

Comment:
@martinpovolny should model class be passed in into this method instead of using EmsCluster for both type of sub menu items, something like this:
def hybrid_infra_name(klass, name1, name2, name3)
          lambda do
            case klass.node_types
            when :non_openstack then name1
            when :openstack     then name2
            else                     name3
            end
          end
        end

        hosts_name    = hybrid_infra_name(EmsCluster,
                                          N_("Hosts"),
                                          N_("Nodes"),
                                          N_("Hosts / Nodes")
        )
        clusters_name = hybrid_infra_name(Host,
                                          N_("Clusters"),
                                          N_("Deployment Roles"),
                                          N_("Clusters / Deployment Roles")
        )